src/calendar/utils/intl.ts

```diff
@@ -29,9 +29,6 @@ export function renderYear(locale: string, date: Date): string {
   });
 }
 
-/*
- `toLocaleDateString` is expensive (10+ ms) to calculate in IE11.
```

Comment:
Yes, `toLocaleDateString` is expensive across all browsers because it creates `DateTimeFormat` object for the specified locale and format arguments, which is expensive operation. Alternatively, we can use `Intl.DateTimeFormat` instead, which allows to create `DateTimeFormat` objects, and cache them in-memory for usage across multiple dates, but for us it makes no difference, because we already use caching object within `getDateLabel` function. So I came to realize that this comment is rather misleading than helpful.